doc/content/help/faq.md

```diff
@@ -1,14 +1,129 @@
+  - `MV2_ENABLE_AFFINITY=0 mpiexec ~/projects/mastodon/mastodon-opt -i shear_beam_Isoil_free_field.i >> out` has the name of the input file. Use the input file name you are interested in.
+
+- Finally, save this file `Example03_hpc.sh` in the folder in which the input file is located. Batch files can be submitted into the queue using the qsub command as shown below.
+
+```
+qsub Example03_hpc.sh
+```
+
+- For more information on PBS see [this](http://hpcweb.inl.gov/home/pbs).
+
+### Some or all my tests are failing, what should I do?
+
+There might be one of several reasons for this:
+
+- You did not compile MASTODON. Compile MASTODON using `make` command.
+
+
+- If the errors while running tests are related to libmesh, your libmesh is out of date. Update and rebuild libmesh using the following command in the MASTODON folder. Then compile MASTODON and try running the tests.
+
+```
+./moose/scripts/update_and_rebuild_libmesh.sh
+```
+
+- You might have accidentally made some changes to MASTODON source code. To undo these changes use the below command. Then update and rebuild libmesh, compile MASTODON, and run the tests as described in the [Getting Started](getting_started/macos_mojave.md) page.  
+
+```
+git clean -dxf
+```
+
+!alert! warning
+Running this command will completely erase all additional files and uncommitted changes to the MASTODON directory.
+!alert-end!
+
+### How to write an input file from scratch?
+
+Generally, we do not recommend doing this. On the [Examples](examples/index.md) page, a variety of problems are discussed. We recommend downloading the input file from one of those problems and modifying it to suit your own problem. In this process, the MASTODON [User Manual](manuals/user/index.md) and [Syntax](syntax/index.md) pages might also help. There are also a number of input files checked into the `tests` directories within MASTODON. In addition, users are welcome to reach out to the MASTODON community through google groups and ask for files similar to what they are doing.
+
+### How to input a mesh file from ABAQUS (i.e., a `.inp` file)?
+
+MASTODON is capable of accepting mesh files in many formats, one of them being an ABAQUS mesh file. Below is an example on how to input an ABAQUS mesh file. Please refer to [`meshgenerators`](/meshgenerators/FileMeshGenerator.md) MOOSE page for more information.
+
+```
+[Mesh]
+  [./file]
+    type = FileMeshGenerator
+    file = ABAQUS_input.inp
+  [../]
+[]
+```
+
+!alert! note
+The .inp reader is currently limited to only one mesh block. If the .inp file contains multiple parts, they will all be moved into one block in MOOSE/MASTODON.
+!alert-end!
+
+### How to specify a velocity or a displacement as the boundary condition? Most example problems use an acceleration boundary condition.
+
+Velocity and displacement can be specified as a boundary condition using [`PresetVelocity`](/bcs/PresetVelocity.md) and [`PresetDisplacement`](/bcs/PresetDisplacement.md), respectively.
```

Comment:
@somu15 the links for these objects shouldn't be HTTP links, but they should be internal links like what I have changed to now. This makes sure that the http link generated in the website is dynamic and will change according to the location of the object. Please follow this in the future.